### PR TITLE
Add basic HTML structure and styles

### PR DESCRIPTION
**Pull request by keya5406**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// Ensure the DOM is fully loaded before running the script
+document.addEventListener('DOMContentLoaded', function() {
+    // Add event listener to the form
+    document.getElementById('passwordForm').addEventListener('submit', function(event) {
+        event.preventDefault(); // Prevent form submission
+
+        // Retrieve the input values
+        const serviceName = document.getElementById('service-name').value.trim();
```
Comment (chintal31) [inline script.js:8]: Nice usage of `trim()`.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        if (username.length < 3) {
+            alert('Username must be at least 3 characters long.');
+            return;
+        }
+
+        // Password validation pattern
+        const passwordPattern = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$/;
+
+        if (!passwordPattern.test(password)) {
+            alert('Password must contain at least one uppercase letter, one lowercase letter, one digit, and one special character.');
```
Comment (chintal31) [inline script.js:28]: Nice usage of regex for password validation. However, I noticed that the error message could be a bit more descriptive. Currently, it doesn’t mention the requirement for a minimum length of 8 characters. You don't need to modify it in this PR but just mentioning for learning purpose.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+footer {
+    background-color: rgba(1, 5, 28, 0.806); /* Semi-transparent background to blend with the image */
+    color: #f5f5f5;
+    text-align: center;
+    padding: 20px 40px;
+    margin-top: auto;
+}
+
+/* Responsive Design */
+@media (max-width: 768px) {
```
Comment (chintal31) [inline styles.css:123]: Nice!!
Did you test this for different screens? If yes, great!! If not, feel free to check how it looks on different screens such as mobile, ipad, desktop etc. using developer tools.

**Review by chintal31 — APPROVED** [review]
Overall, the webpage looks good! 👍

It seems like you’ve gained a good understanding of:

- HTML tags and their appropriate use in structuring different sections of the page.
- CSS properties and how they can be applied to style elements effectively.
- JavaScript interactivity to enhance user experience.
- Core concepts like trim(), regex for validation, and media queries for responsive design.

If any of these concepts feel unclear, I recommend you to dive deeper into them—they’re essential skills for building robust and user-friendly web applications.

**Review by Atharva0418 — APPROVED** [review]

**Pull request merged**